plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/DeployConsole.java

```diff
@@ -0,0 +1,33 @@
+package com.google.cloud.tools.eclipse.appengine.deploy.ui;
+
+import org.eclipse.ui.console.MessageConsole;
+
+import com.google.cloud.tools.eclipse.appengine.deploy.standard.StandardDeployJob;
+import com.google.cloud.tools.eclipse.ui.util.MessageConsoleUtilities.ConsoleFactory;
+
+public class DeployConsole extends MessageConsole {
```

Comment:
It would also be useful to update the console description with the job state. For example, a terminated launch shows "<terminated> gcloud [Eclipse Application] …" when dead. (We should do the same with the server launch console too)